### ISSUE TITLE
Failed to execute 'removeChild' on 'Node'

### ISSUE THREAD

**Issue by basz** [kaliber5/ember-stargate#11]
I have a template containing

```
<PortalTarget @name="target" class="btn-group pr-1 pb-1"/>

{{#if this.editMode}}
  {{component ''my-edit-component"}}
{{else}}
  {{component ''my-view-component"}}
{/if}
```

both components Portal into the same PortalTarget... When I toggle this.editMode I get the following error...

Fatal error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node. I remember a similar thing was the case with ember-wormhole. Conditionals inside the sending component needed to be wrapped inside a div to avoid this error if I remember correctly...

**Comment by simonihmig** [comment]
Yeah, I remember this problem. Not sure if that's the case here. But the error definitely sounds like something failing inside GlimmerVM. Are you on the latest Ember?
